### PR TITLE
Add jsonld context for application profile

### PR DESCRIPTION
**Pull request by justinwb**
Adding a jsonld context specifically for the [Application Profile](https://solid.github.io/data-interoperability-panel/specification/#app), which needs to be able to cross-pollinate with the contents of the [Solid-OIDC Client Identifier Document](https://solid.github.io/solid-oidc/#clientids-document) (see https://github.com/solid/data-interoperability-panel/issues/210).

There are a couple of determinations we still need to make which would adjust the contents of the `@context` in this PR:

- Rather than including `application_name`/`interop:applicationName` in this context, should we instead use the value from `oidc:client_name`? 
- Rather than including `application_thumbnail`/`interop:applicationThumbnail` in this context, should we instead use the `oidc:logo_uri`?
- Would `oidc:contacts` be equivalent to `application_author`/`interop:applicationAuthor`?

Personally I lean towards DRY and feel we should reuse what's already being specified in solid-oidc and extend

### CODE REVIEW

**Comment by elf-pavlik** [comment]
Looking at https://openid.net/specs/openid-connect-registration-1_0.html#ClientMetadata

> `logo_uri` - OPTIONAL. URL that references a logo for the Client application. If present, the server SHOULD display this image to the End-User during approval. The value of this field MUST point to a valid image file.

> `client_name` - OPTIONAL. Name of the Client to be presented to the End-User.

> `contacts` - OPTIONAL. Array of e-mail addresses of people responsible for this Client. This might be used by some providers to enable a Web user interface to modify the Client information. 

Since `contacts` is defined as array of e-mail addresses we probably don't want to use it. The other two seem appropriate to me.

File: proposals/specification/interop-application.jsonld [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "@id": "http://www.w3.org/ns/solid/interop#applicationDescription"
+    },
+    "application_author": {
+      "@id": "http://www.w3.org/ns/solid/interop#applicationAuthor",
+      "@type": "@id"
+    },
+    "application_thumbnail": {
+      "@id": "http://www.w3.org/ns/solid/interop#applicationThumbnail",
+      "@type": "@id"
+    },
```
Comment (elf-pavlik) [inline proposals/specification/interop-application.jsonld:18]:
```suggestion
```

File: proposals/specification/interop-application.jsonld [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+    "application_name": {
+      "@id": "http://www.w3.org/ns/solid/interop#applicationName"
+    },
```
Comment (elf-pavlik) [inline proposals/specification/interop-application.jsonld:7]:
```suggestion
```

**Review by elf-pavlik — APPROVED** [review]

**Comment by justinwb** [comment]
> The other two seem appropriate to me.

👍  Agree - so now the question becomes whether we should remove those terms from the interop vocabulary, adjust the current spec to reflect the terms we are using, etc. Thoughts?

**Pull request merged**